jx/bdd/static/jx-requirements.yml

```diff
@@ -0,0 +1,46 @@
+gitops: true
```

Comment:
@MarckK, what are you trying to do? Making a "non Boot" cluster to use Boot?

Reply:
This was just me trying to push things around last weekend. Don't worry, I've already rolled this back locally. Am trying a new approach, will push up this weekend.

Reply:
No problem at all :-) I was just wondering ;-)